### ISSUE TITLE
InvalidQuery exception raised when attempting to export entry from a 'DB store' handler

### ISSUE THREAD

**Issue by ababic** [barseghyanartur/django-fobi#44]
Am seeing this error on the Heroku demo project when trying to export entries for any form. For example: 

- https://django-fobi.herokuapp.com/en/fobi/forms/edit/857/
- https://django-fobi.herokuapp.com/en/fobi/forms/edit/584/

**Comment by ababic** [comment]
Sorry to jump right in there with an issue without saying thank you for all the work you've put in with this app. I'm checking it out for use in a project, and it's looking like it will be super useful. The codebase is in great shape, too.

Also noticed another issue when submitting any of the forms in the demo with Mail handlers applied, but I imagine that might be on purpose to prevent abuse?

**Comment by barseghyanartur** [comment]
@ababic:

Could you post some screenshots and traces perhaps? I, as a test_user (see the readme for credentials), can export entries (just tested for both forms).

Regarding the mail handler - it's has to do with the fact, that google changed their API some time ago. At the moment a test account is used to send emails, but they are being blocked by Google. Therefore, the exception. With proper settings, problem would not exist.

**Comment by ababic** [comment]
Thanks for the responding so quickly :)

**Comment by barseghyanartur** [comment]
@ababic:

Fixed in 0.9.6.
